js/bruh-dash.js

```diff
@@ -1,83 +1,209 @@
 var global = window || GLOBAL;
 
 global.bruhdash = {
-  chunk: function(){
-
-  },
-
-  compact: function() {
-
-  },
-
-  difference: function() {
-
-  },
-
-  drop: function(){
-
-  },
-
-  dropRight: function() {
-
-  },
-
-  fill: function() {
-
-  },
-
-  first: function () {
-      
-  },
-
-  indexOf: function () {
-
-  },
-
-  inital: function () {
-
-  },
-
-  last: function () {
-
-  },
-
-  lastIndexof: function () {
-
-  },
-
-  pull: function () {
-
-  },
-
-  pullAt: function () {
-
-  },
-
-  rest: function () {
-
-  },
-
-  slice: function () {
-
-  },
-
-  take: function () {
-
-  },
-
-  takeRight: function () {
-
-  },
-
-  zip: function () {
-
-  },
-
-  unzip: function () {
-
-  },
-
-  without: function() {
-
+  chunk: function(array, size){
+   var chunks;
+   var resultArray = [];
+   if(size < 1 || isNaN(size)){
+     size = 1;
+   }
+   while(array.length){
+      chunks = array.splice(0,size);
+      resultArray.push(chunks);
+   }
+
+   return resultArray;
+  },
+
+  compact: function(array) {
+    for(var i = 0; i < array.length; i++){
+      if(array[i] === false || array[i] === null || array[i] === 0
```

Comment:
This is good. Though, you can leverage the truthiness value of each element to make this condition a little more succinct.